src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Navigation.java

```diff
@@ -595,10 +597,11 @@ private void addHelpLink(Content tree) {
     }
 
     private void addSearch(Content tree) {
-        String search = "search";
         String reset = "reset";
-        HtmlTree inputText = HtmlTree.INPUT("text", HtmlIds.SEARCH_INPUT, search);
-        HtmlTree inputReset = HtmlTree.INPUT(reset, HtmlIds.RESET_BUTTON, reset);
+        HtmlTree inputText = HtmlTree.INPUT("text", HtmlIds.SEARCH_INPUT)
+                .put(HtmlAttr.PLACEHOLDER, searchPlaceholder);
+        HtmlTree inputReset = HtmlTree.INPUT(reset, HtmlIds.RESET_BUTTON)
+                .put(HtmlAttr.VALUE, reset);
```

Comment:
Isn't it a mere coincidence that "reset" is both an input type and the value of the "value" attribute: 
```
<input type="reset" id="reset-button" disabled value="reset">
```
Why do we need this "value" attribute?